### ISSUE TITLE
How to inference on a image or video?

### ISSUE THREAD

**Issue by Suncheng2022** [zhenyuw16/UniDetector#4]
Hello, how can I get the detection results on pictures or videos, I really hope to see the detection results of new categories! 
But I don't know how to run it. Can I refer to the "demo" folder?
Hope to get your guidance, thank you!

**Comment by Suncheng2022** [comment]
The "demo" does not seem to be the function implemented in this article, it seems to be the function of mmcv, right?

**Comment by zhenyuw16** [comment]
Yes, we have not updated the "demo" folder yet. Now if you want to get the detection results, you can first train a model, then inference it on the images you have, following the inference config file.

**Comment by Suncheng2022** [comment]
> Yes, we have not updated the "demo" folder yet. Now if you want to get the detection results, you can first train a model, then inference it on the images you have, following the inference config file.

Emmm, Sorry, I encountered something wrong when preparing the dataset. #5

**Comment by HanJW2000** [comment]
> > Yes, we have not updated the "demo" folder yet. Now if you want to get the detection results, you can first train a model, then inference it on the images you have, following the inference config file.
> 
> Emmm, Sorry, I encountered something wrong when preparing the dataset. #5

Do you slove it？I also cannot inference on a image

**Comment by feature-space-move** [comment]
> > > Yes, we have not updated the "demo" folder yet. Now if you want to get the detection results, you can first train a model, then inference it on the images you have, following the inference config file.
> > 
> > 
> > Emmm, Sorry, I encountered something wrong when preparing the dataset. #5
> 
> Do you slove it？I also cannot inference on a image

I don't think the author has released this part of the code, and this part is exactly the selling point of the UNIDETECTOR. I don’t know when it will be open source ... emmm, you can take a look at Microsoft's GLIP related work, at least the GLIP open source all the code, but you need to make some modification yourself for your need.

**Comment by WangPingA** [comment]
请问您复现出来了吗？可以请教一些问题吗，谢谢
